style.css

```diff
@@ -579,3 +621,317 @@ label {
   position: absolute;
   margin-left: 74px;
 }
+
+@media (min-width: 768px) {
+  body {
+    background-color: #e5e5e5e5;
+  }
+
+  .desktop-nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 50px;
+  }
+
+  .loose {
+    display: none;
+  }
+
+  .main-section {
+    background-image: url(images/desktop-header.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #fff;
+    border-bottom-left-radius: 90px;
+    height: 100vh;
+  }
+
+  .main-section h1 {
+    height: auto;
+    width: 900px;
+    font-size: 48px;
+  }
+
+  .main-section p {
+    width: 800px;
+    height: auto;
+    margin-right: 90px;
+    margin-top: -5px;
+  }
+
+  .main-section h3 {
+    height: auto;
+    padding: 5px;
+    font-size: 16px;
+    margin-right: 700px;
+  }
+
+  .nav-links {
+    padding: 8px;
+    margin-right: 670px;
+  }
+
+  .projects {
+    display: grid;
+    grid-template-columns: 90%;
+    margin-top: 80px;
+    max-width: 100%;
+    column-gap: 30px;
+  }
+
+  .work {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    background-color: #fff;
+  }
```

Comment:
- [ ] Kindly make sure each project card looks similar to the given [Figma design 🎨](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=aBYhrF8gMscZ1eny-0) as I've noticed some spacing issues on your design:
    - Your design:
    ![firefox_tjejGVxT62](https://user-images.githubusercontent.com/104968236/228869055-b68f3f26-52c4-40e4-b366-1e24b7cd972b.png)
    - Figma design:
      ![firefox_LKKVzuss0t](https://user-images.githubusercontent.com/104968236/228869267-3207c258-914f-46bc-aac2-ce26a3f88ce7.png)
    - As I've highlighted in yellow, you can see the spacing is incorrect, so kindly fix that.
- [ ] Please fix the spacing issue with project cards with the description to the left. (this applies to all cards that have the same layout)
     - Your design:
       ![firefox_t0ue9vqBp5](https://user-images.githubusercontent.com/104968236/228871909-028ff112-eea7-4cda-b28c-4edce2f7ecf2.png)
    - Figma design:
      ![firefox_4qsdzkLYDM](https://user-images.githubusercontent.com/104968236/228871990-05687774-2b6d-46ae-bf95-d4642c7083d1.png)

 - [ ] Once again please fix this spacing issue between the project cards:
     - Your design:
       ![firefox_Vs4iNFs5gH](https://user-images.githubusercontent.com/104968236/228870372-d2ce928d-56e7-48df-8800-35a5e8ba3615.png)
    - Figma design:
      ![ApplicationFrameHost_kxUI1qEOAh](https://user-images.githubusercontent.com/104968236/228870448-dd3e8056-fa04-4016-b386-b73fe5eddb26.png)